workflow/scripts/export_ariadne_variables.py

```diff
@@ -3898,17 +3902,69 @@ def get_export_import_links(n, region, carriers):
     #     exports_h2 * MWh2PJ
 
     # Trade|Secondary Energy|Liquids|Hydrogen|Volume
+
+    renewable_oil_supply = (
+        n.statistics.supply(bus_carrier="renewable oil", **kwargs)
+        .groupby(["bus", "carrier"])
+        .sum()
+    )
+
+    assert region == "DE"  # only DE is implemented at the moment
+
+    DE_renewable_oil = renewable_oil_supply.get("DE renewable oil", pd.Series(0))
+    EU_renewable_oil = renewable_oil_supply.get("EU renewable oil", pd.Series(0))
+
+    if DE_renewable_oil.sum() == 0:
+        DE_bio_fraction = 0
```

Comment:
Rather renewable fraction? Or is it only from biomass sources?

Reply:
only bio sources